tests/Unit/LoanServiceTest.php

```diff
@@ -208,7 +208,7 @@ public function testServiceCanRepayAScheduledRepaymentConsecutively()
             'amount' => 1667,
             'outstanding_amount' => 0,
             'currency_code' => $currencyCode,
-            'due_date' => '2020-02-20',
+            'due_date' => '2020-04-20',
```

Comment:
`due_date` when creating the `$scheduledRepaymentThree` object was passed as `2020-04-20` : https://github.com/R4CHI7/laravel-code-challenge-2/blob/main/tests/Unit/LoanServiceTest.php#L182